fiftyone/utils/eta.py

```
@@ -43,6 +43,77 @@
+                        detections["output_1"],
+                        detections["output_2"],
+                    )
+
+            return predict
+
+        tfmodels._load_tf2_detection_model = _load_tf2_detection_model_fixed
+        _tf2_patched = True
+
+    except ImportError:
+        pass
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)